example.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use PasswordPolicy\Policy;
+
+require 'vendor/autoload.php';
+
+$policy = (new Policy())
```

Comment:
Library has PHP 5.3 as minimal requirement, but this example will work only starting from PHP 5.4.